packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -93,76 +158,107 @@ export const makeZoeHelpers = (zcf) => {
     rejectOffer,
 
     /**
-     * Compare two proposals for compatibility. This returns true
-     * if the left offer would accept whatever the right offer is offering,
-     * and vice versa.
-     *
-     * @param {OfferHandle} leftOfferHandle
-     * @param {OfferHandle} rightOfferHandle
-     * @returns boolean
-     *
+     * Check whether an update to currentAllocation satisfies
+     * proposal.want. Note that this is half of the offer safety check
+     * - whether the allocation is a refund is not checked. Allocation
+     * is merged with currentAllocation, overwriting if the keywords
```

Comment:
Ok, that sounds good. Is there anything you would change in the above text?

Reply:
> - whether the allocation is a refund is not checked. Allocation is merged with currentAllocation, overwriting if the keywords are the same to produce the newAllocation

"whether the allocation constitutes a refund is not checked. Allocation is merged with currentAllocation (allocations' values prevailing if the keywords are the same) to produce the newAllocation."

I changed "is" to "constitutes" because something my qualify as a refund while giving more back. Since that's _not_ what we're checking here, it may be too fine a point. 

I changed to the parenthetical to make the reading that we're overwriting (modifying) values in one of the inputs less plausible.

Reply:
Done